cpp/ql/lib/semmle/code/cpp/rangeanalysis/new/internal/semantic/analysis/RangeAnalysisSpecific2.qll

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
I think you mentioned you wanted to call this file something more ... meaningful at some point? 😄

Reply:
Yeah... I never did find the right name for it. Maybe `RangeAnalysisSpecificRelative`?

Reply:
That sounds good to me 🙂.